src/Tasks/Microsoft.NET.Build.Tasks/build/Microsoft.Packaging.Tools.targets

```diff
@@ -0,0 +1,133 @@
+<!--
+***********************************************************************************************
+Microsoft.Packaging.Tools.targets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved. 
+***********************************************************************************************
+-->
+
+<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+  </PropertyGroup>
+  
+  <Choose>
+    <!-- Allow completely disabling the conflict resolution targets-->
+    <When Condition="'$(DisableHandlePackageFileConflicts)' == 'true'" />
+    <!-- Condition here is a hack until https://github.com/dotnet/sdk/issues/534 is fixed -->
+    <When Condition="'$(TargetFramework)' != '' or '$(TargetFrameworks)' != ''">
+      <!-- NuGet 4, run after the targets that construct items from deps -->
+      <PropertyGroup>
+        <HandlePackageFileConflictsAfter>ResolvePackageDependenciesForBuild</HandlePackageFileConflictsAfter>
+        <HandlePublishFileConflictsAfter>RunResolvePublishAssemblies</HandlePublishFileConflictsAfter>
+        <RemoveDepsFileConflictsAfter>GenerateBuildDependencyFile</RemoveDepsFileConflictsAfter>
+        <RemovePublishDepsFileConflictsAfter>GeneratePublishDependencyFile</RemovePublishDepsFileConflictsAfter>
+        <!-- If CopyLocalLockFileAssemblies is not true the host will run from packages,
+             but we still need to consider those package items for conflicts -->
+        <HandlePackageFileConflictsDependsOn Condition="'$(CopyLocalLockFileAssemblies)' != 'true'">_GetLockFileAssemblies</HandlePackageFileConflictsDependsOn>
+
+        <_supportsTrimming>true</_supportsTrimming>
+      </PropertyGroup>
+    </When>
+    <When Condition="'$(ResolveNuGetPackages)' == 'true' and Exists('$(ProjectLockFile)')">
+      <!-- NuGet 3, run after the target that constructs items from lock file -->
+      <PropertyGroup>
+        <HandlePackageFileConflictsAfter>ResolveNuGetPackageAssets</HandlePackageFileConflictsAfter>
+      </PropertyGroup>
+    </When>
+    <Otherwise>
+      <!-- NuGet 2, run before targets that consume references -->
```

Comment:
I don't think we need to handle NuGet 2 or 3.  We will only ever support the NuGet we are using.